### ISSUE TITLE
[1.1] [BUG] Migration package not compatible on lumen 5.2

### ISSUE THREAD

**Issue by boedy** [laravel-doctrine/orm#164]
### Lumen 5.2. | ORM 1.1

**Packages:**
"php": ">=5.5.9",
"laravel/lumen-framework": "5.2._",
"vlucas/phpdotenv": "~2.2",
"laravel-doctrine/orm" : "1.0._",
"laravel-doctrine/migrations" : "1.0.*",

**Fatal Error**

```
  [Symfony\Component\Debug\Exception\FatalThrowableError]
  Parse error: syntax error, unexpected '\' (T_NS_SEPARATOR), expecting identifier (T_STRING)
```
### Steps to reproduce the behaviour

The following commands are broken:
- doctrine:migrations:migrate
- doctrine:migrations:rollback

They throw the above error.

**Comment by patrickbrouwers** [comment]
Tested on Lumen 5.2, cannot reproduce. I'm afraid this syntax error is in your own code.

**Comment by josenicomaia** [comment]
Could it be your PHP version?

**Comment by boedy** [comment]
Weird. Got resolved after downgrading 1.0 => upgrading 1.1.

Probably some 3rd party package which broke it.
